### ISSUE TITLE
给 eva title 截止了 字体的 fallback

### ISSUE THREAD

**Issue by itorr** [itorr/itorr#6]
和小洪聊天的时候 提到了一个 https://zfont.cn/ 的网站

在这个网站上看到了 半年前我构想的一个 截止 字体 fallback 方案被应用了，看来这个方法是可行的

我把自己在用的字体，放上去发现并没有正常工作

但是换用 zfont 网站上的演示又可以工作，怀疑是 zfont 在用的 这个截止字体字型不全
于是自己打开 Glyphs 3 新建了覆盖了英中日韩常见符号字型的空白字体

测试一下可以正常工作，完整覆盖了缺失中文字型的常见日文字体，记录里面的所有缺失字型都可以覆盖到，特别妙ヽ(･ω･´ﾒ)

就是这个字体文件体积有点大，压缩成 woff2 之后还要 110KB，gzip 之后 86.3KB 勉强可以接受（个屁

再试试把字体按照 TrueType 导出一份 ttf，使用 Google 的 woff2 字体转换库 压缩一份 woff2 试试

这次是 78.4KB Gzip 之后是 63.7KB 开始可以接受了 (･ェ･。)

感觉如果深入到字体内看下数据结构 应该还能把体积缩的更小，现在 空字型居然都可以 63.7KB 这个尺寸太离谱了

嘛，不过总比方案不可行要好ψ(｀∇´)ψ

给 eva-title 更新上了
https://github.com/itorr/eva-title/issues/31

现在不存在字型的文字会留空啥也不绘制，不会 fallback 到什么都没有的字体上啦

**Comment by LANMIN-X** [comment]
是的 我就是站长，我只包括gb2312和常用日文英文繁体字，为了减少服务器压力，因为动态裁剪子集不会弄，还有你说的fallback方案就是为了不支持的字形上他就显示空白，为了更好区别字形的支持。

**Comment by itorr** [comment]
> 是的 我就是站长，我只包括gb2312和常用日文英文繁体字，为了减少服务器压力，因为动态裁剪子集不会弄，还有你说的fallback方案就是为了不支持的字形上他就显示空白，为了更好区别字形的支持。

是很好的方案！zfont 也很好用ww

后来把 blank-font 单开了一个仓库维护 https://github.com/itorr/blank-font/ 实际上需要补充的文字还很多，可能还是要想办法通过程序生成，没想到什么更好的办法
